ui/src/VoxettaApp.js

```diff
@@ -30,3 +26,36 @@ export class VoxettaApp extends LitElement {
 }
 
 customElements.define('vox-app', VoxettaApp);
+
+
+navigator.mediaDevices.getUserMedia({ audio: true, video: false })
+    .then(function(stream) {
+
+        const recordButton = document.getElementById("recordButton");
+        const audioSave = document.getElementById("utterance");
+        let isRecording = false;
+        let recordedChunks = [];
+        const mediaRecorder = new MediaRecorder(stream);
+        
+        recordButton.addEventListener('click', (ev)=>{
+            if(!isRecording){
+                isRecording = true;
+                mediaRecorder.start();
+            }else{
+                isRecording = false;
+                mediaRecorder.stop();
+                audioSave.style.display = "block";
+            }
+        })
+
+        mediaRecorder.ondataavailable = function(ev) {
+            recordedChunks.push(ev.data);
+        }
+
+        mediaRecorder.onstop = (ev)=>{
+            let blob = new Blob(recordedChunks, { 'type' : 'audio/mp3;' });
```

Comment:
@everyone Do we know what sort of audio file is preferred? I believe Blobstore should be able to handle all of the basic types